sections/dashboard/MobileDashboard/MobileDashboard.tsx

```diff
@@ -5,14 +5,30 @@ import { CompetitionBanner } from 'sections/shared/components/CompetitionBanner'
 import FuturesMarkets from './FuturesMarkets';
 import OpenPositions from './OpenPositions';
 import Portfolio from './Portfolio';
+import SpotMarkets from './SpotMarkets';
 
-const MobileDashboard: React.FC = () => (
-	<div>
-		<CompetitionBanner />
-		<Portfolio />
-		<OpenPositions />
-		<FuturesMarkets />
-	</div>
-);
+enum PositionsTab {
+	FUTURES = 'futures',
+	SPOT = 'spot',
+}
+
+const MobileDashboard: React.FC = () => {
+	const [activePositionsTabInParent, setActivePositionsTabInParent] = React.useState<PositionsTab>(
```

Comment:
Instead of tracking the `activePositionsTab` with two independent pieces of state (the other one is in the `OpenPositions` component), we should consider trying to condense them into one, or even better, move the state into Recoil, so we can share it between the desktop and mobile versions.